examples/plot_hawkes_cumulants_matching.py

```diff
@@ -0,0 +1,55 @@
+"""
+=======================================
+Fit Hawkes kernel norms using cumulants
+=======================================
+
+This non parametric Hawkes cumulants matching
+(`tick.inference.HawkesCumulantMatching`) algorithm estimates directly
```

Comment:
`tick.hawkes.HawkesCumulantMatching`

Reply:
Damn... I'll force push with this and merge directly.

You want an example on real data? Because I have already provided an example on simulated data.